### PR TITLE
Update default Renderer to expose Image.

### PR DESCRIPTION
**Pull request by SolarLune**
Renamed Renderer.img to Renderer.Result for this to be clearer.
Cleaned up Readme and added more info.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,27 +21,46 @@ import (
...
+	// Renderer's Save functions.
+
 }
 
 ```
 
 ## Documentation
 
-For docs, see https://godoc.org/github.com/lafriks/go-tiled or run:
+For further documentation, see https://godoc.org/github.com/lafriks/go-tiled or run:
```
Comment (lafriks) [inline README.md:63]:
```suggestion
For further documentation, see https://pkg.go.dev/github.com/lafriks/go-tiled or run:
```

File: render/renderer.go [hunk 2]
```diff
@@ -213,17 +213,24 @@ func (r *Renderer) RenderVisibleLayers() error {
 	return nil
 }
 
+// Clears the render result to allow for separation of layers. For example, you can
```
Comment (lafriks) [inline render/renderer.go:216]:
```suggestion
// Clear the render result to allow for separation of layers.
// For example, you can
```

**Comment by lafriks** [comment]
See drone ci lint for more details

**Comment by SolarLune** [comment]
Got it; I've applied those changes. Thanks for reviewing it!

**Comment by lafriks** [comment]
Thanks for your PR

**Comment by SolarLune** [comment]
No problem! Thanks for the review.

**Pull request merged**